src/main/java/com/google/sps/GraphGenerationTest.java

```diff
@@ -111,7 +113,7 @@ public void correctProtoToGraphWithDataAndMetadata() {
         Struct.newBuilder().putFields("row", rowValue).putFields("column", colValue).build();
     nodeA.setMetadata(metadata);
 
-    GraphNode graphNode = Utility.protoNodeToGraphNode(nodeA.build());
+    GraphNode graphNode = protoNodeToGraphNode(nodeA.build());
     List<String> tokenList = graphNode.tokenList();
     Struct generatedMetadata = graphNode.metadata();
```

Comment:
I noticed in the `correctProtoToGraphNoData` function you still reference `Utility.protoNodeToGraphNode(nodeA.build());`. Accident?